ct/openwebui.sh

```diff
@@ -55,23 +55,57 @@ function default_settings() {
 }
 
 function update_script() {
-header_info
 if [[ ! -d /opt/open-webui ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
-msg_info "Updating ${APP} (Patience)"
-cd /opt/open-webui
-output=$(git pull --no-rebase)
-if echo "$output" | grep -q "Already up to date."
-then
-  msg_ok "$APP is already up to date."
-  exit
+UPD=$(whiptail --backtitle "Proxmox VE Helper Scripts" --title "SUPPORT" --radiolist --cancel-button Exit-Script "Spacebar = Select" 11 58 2 \
+  "1" "Update ${APP}" ON \
+  "2" "Install Ollama" OFF \
+  3>&1 1>&2 2>&3)
+
+header_info
+if [ "$UPD" == "1" ]; then
+  msg_info "Updating ${APP} (Patience)"
+  cd /opt/open-webui
+  output=$(git pull --no-rebase)
+  if echo "$output" | grep -q "Already up to date."
+  then
+    msg_ok "$APP is already up to date."
+    exit
+  fi
+  systemctl stop open-webui.service
+  npm install &>/dev/null
+  npm run build &>/dev/null
+  cd ./backend
+  pip install -r requirements.txt -U &>/dev/null
+  systemctl start open-webui.service
+  msg_ok "Updated Successfully"
+fi
+if [ "$UPD" == "2" ]; then
+  msg_info "Installing Ollama"
+  curl -fsSLO https://ollama.com/download/ollama-linux-amd64.tgz
+  tar -C /usr -xzf ollama-linux-amd64.tgz
+  rm -rf ollama-linux-amd64.tgz
+  cat <<EOF >/etc/systemd/system/ollama.service
+[Unit]
+Description=Ollama Service
+After=network-online.target
+
+[Service]
+Type=exec
+ExecStart=/usr/bin/ollama serve
+Environment=HOME=$HOME
+Environment=OLLAMA_HOST=0.0.0.0
+Restart=always
+RestartSec=3
+
+[Install]
+WantedBy=multi-user.target
+EOF
+  systemctl enable -q --now ollama.service
+  sed -i 's/ENABLE_OLLAMA_API=false/ENABLE_OLLAMA_API=true/g' /opt/open-webui/.env
```

Comment:
These env changes to `open-webui` will not overwrite settings stored in database, so user has to manually enable Ollama if they have touched the setting before. Actually i will set `OLLAMA_BASE_URL` in install script instead, so the user does not have to figure out the ollama ip themselves

Reply:
https://github.com/tteck/Proxmox/compare/b04a739828cb0277c71d8003b4e91d417c8113d4..64b0e0121a12e3130ad8eab5244a672d859ac834

Reply:
Then maybe use 
```
read -r -p "Would you like to add Ollama? <y/N> " prompt
if [[ ${prompt,,} =~ ^(y|yes)$ ]]; then
```
instead of adding it to the whiptail

Reply:
Ah nice, will do

Reply:
I meant the` openwebui-install.sh`